src/styles/blocks/catalog.scss

```
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(2, $card-width);
+  justify-content: center;
+  gap: 48px;
+  min-width: 488px;
+
+  &__card {
+    display: flex;
```

```suggestion
    display: grid;
```
По завданню ми мала замінити флекси на гріди

---

Для каталога и есть грид. Флекс для надписей в карточках, там нет смысла грид выставлять для одной фразы.